test/TestProjects/Lro-Basic-TypeSpec/Lro-Basic-TypeSpec.tsp

```diff
@@ -43,17 +43,19 @@ model Thing {
 interface Authoring {
   createProjectStatusMonitor is CustomCore.PollingOperation<Project>;
 
-  @convenientAPI(true)
-  @Azure.Core.pollingOperation(Authoring.createProjectStatusMonitor)
-  createProject is Azure.Core.LongRunningResourceCreateWithServiceProvidedName<Project>;
-  
+// uncomment the following after https://github.com/Azure/typespec-azure/issues/3109 is fixed
+//   @convenientAPI(true)
+//   @Azure.Core.pollingOperation(Authoring.createProjectStatusMonitor)
+//   createProject is Azure.Core.LongRunningResourceCreateWithServiceProvidedName<Project>;
   @convenientAPI(true)
   @Azure.Core.pollingOperation(Authoring.createProjectStatusMonitor)
```

Comment:
Actually this pollingOperation is not valid, and it will not affect the LRO definition. It is `LongRuningResource` which define the LRO metedata. We need to redefine the test case.